mods/weather/init.lua

```diff
@@ -88,7 +88,8 @@ local function update_clouds()
 	local n_speedz = nobj_speedz:get_2d({x = time, y = 0}) -- -1 to 1
 
 	for _, player in ipairs(minetest.get_connected_players()) do
-		local humid = minetest.get_humidity(player:get_pos())
+		-- Default to 50 when no biomes are available
```

Comment:
This comment seems unsuitable, humidity noise exists independent of biomes. Better is:
`-- Fallback to 50 for very old worlds`

Reply:
Humidity requires the biome noise params to be present, hence this wording. Will change it according to your suggestion.